spec/controllers/tasks_controller_spec.rb

```diff
@@ -25,28 +46,37 @@
             expect(response).to have_http_status(:success)
         end
 
-        it "retuns body with 2 tasks" do
-            Task.create(description: "walk the dog")
-            Task.create(description: "feed the dog")
+        it "retuns body with 1 tasks" do
```

Comment:
Also the presence of this tests makes me think I should see a test case to return multiple tasks.

Reply:
added